JDBC/DeleteBooks.java

```diff
@@ -0,0 +1,52 @@
+package com.metacube.jdbc.example;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+public class DeleteBooks {
+	public static void main(String[] args) throws IOException {
+		getDeletedBooks();
+		System.out.println(DeleteBookPOJO.getNumberOfBooksDeleted()+" Book Deleted Successfully");
+	}
+	/* execute query using prepared statement*/
+	/*method to get number of books deleted*/
+	private static boolean getDeletedBooks() {
```

Comment:
ACTION: getDeletedBooks() function sounds like it will return records of deleted books but it is not returning a Boolean value, Either return list of deleted books or change te function name